src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -230,7 +230,7 @@ public function addXmlContent($content, $charset = 'UTF-8')
         $dom->validateOnParse = true;
 
         if ('' !== trim($content)) {
-            @$dom->loadXML($content, LIBXML_NONET);
+            @$dom->loadXML($content, LIBXML_NONET | LIBXML_PARSEHUGE);
```

Comment:
@zorn-v is it always defined, or does it depend on the libxml version being used ? Distributions generally compile PHP against the system libxml rather than the version bundled with PHP, meaning that it may change